lib/test/helpers/util/utils.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+function isInteger(x) {
```

Comment:
Why not just https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger ?

Reply:
updated